### PR TITLE
Closes softlayer/sl-ember-components#544

### PR DESCRIPTION
**Pull request by juwara0**
Wrote integration tests for sl-alert component
Deleted unit tests for sl-alert component

### CODE REVIEW

File: tests/integration/components/sl-alert-test.js [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+        this.$( '.sl-alert button' ).length,
+        1,
+        'Dismissable button was rendered'
+    );
+});
+
+test( 'Dismiss Action is called on button click', function( assert ) {
+
+    const dismissAction = () => {
+        assert.ok( 'A dissmiss action was called' );
```
Comment (notmessenger) [inline tests/integration/components/sl-alert-test.js]: Spelling of "dissmiss"

File: tests/integration/components/sl-alert-test.js [hunk 2]
```diff
@@ -0,0 +1,137 @@
...
+    };
+
+    this.render( hbs`
+        {{#sl-alert dismissable=true dismiss='dismissAction'}}
+            Dissmissable is true with dismiss action
```
Comment (notmessenger) [inline tests/integration/components/sl-alert-test.js]: Spelling of "dissmiss"

File: tests/integration/components/sl-alert-test.js [hunk 3]
```diff
@@ -0,0 +1,137 @@
...
+        {{/sl-alert}}
+    ` );
+
+    const $button = this.$( '.sl-alert button' );
+    this.on( 'dismissAction', dismissAction );
+    $button.click();
+});
+
+test( 'Dismiss Action is not possible when dissmissable is false', function( assert ) {
```
Comment (notmessenger) [inline tests/integration/components/sl-alert-test.js]: Spelling of "dissmissable"

File: tests/integration/components/sl-alert-test.js [hunk 4]
```diff
@@ -0,0 +1,137 @@
...
+
+    this.render( hbs`
+        {{#sl-alert dismiss='dismissAction'}}
+            Dissmissable is false with dismiss action
```
Comment (notmessenger) [inline tests/integration/components/sl-alert-test.js]: Spelling of "dissmissable"

File: tests/integration/components/sl-alert-test.js [hunk 5]
```diff
@@ -0,0 +1,137 @@
...
+    ` );
+
+    assert.equal(
+        this.$( '.sl-alert button' ).length,
+        0,
+        'Dismissable button was not rendered'
+    );
+
+    assert.strictEqual(
+        this.$( '.sl-alert' ).hasClass( 'alert-dismissable' ),
```
Comment (notmessenger) [inline tests/integration/components/sl-alert-test.js]: Should this class be checked *for* when dismissable is true?

Comment (juwara0) [inline tests/integration/components/sl-alert-test.js]: Good idea. I am adding that check in.

**Pull request merged**